docs/setup/create-repo.md

```diff
@@ -16,41 +16,53 @@ It can be considered the lakeFS analog of a bucket in an object store. Since it
 
 ## Create the first user
 
-When you first open the lakeFS UI, you will be asked to create an initial admin user.
```

Comment:
I think this should stay, in addtition to the Note below. It is much more welcoming

Reply:
Done